Stepippo/Classes/Controllers/GoalSettingVC.swift

```diff
@@ -24,6 +22,14 @@ final class GoalSettingVC: UIViewController {
     }
     
     @IBAction func stockButton(_ sender: Any) {
+        
+        if addTaskTextField.text != nil && addTaskTextField.text != "" {
```

Comment:
```suggestion
        if let inputText = addTaskTextField.text, !addTaskTextField.text.isEmpty {
```
のように書くと、if の波括弧内で今日西安ラップの必要なく、`inputText`という名前で使えるようになります👌
空文字判定はisEmptyが使えますね🙆‍♂️

Reply:
このあたりから #199 の領域に入ってます(^_^;)

Reply:
はい、ちょっと入っちゃいましたね👌🤔